src/server/auth/server/config.go

```diff
@@ -517,7 +517,6 @@ func (a *apiServer) setCacheConfig(config *auth.AuthConfig) error {
 	for _, idp := range newConfig.IDPs {
 		if idp.SAML != nil {
 			a.samlSP = &saml.ServiceProvider{
-				Logger:      logrus.StandardLogger(),
```

Comment:
They've removed the `logger` from ServiceProvider because they no longer log errors during validation.